.eslintrc.json

```diff
@@ -9,18 +9,21 @@
       },
       "extends": ["plugin:@angular-eslint/recommended"],
       "rules": {
+        "@angular-eslint/no-empty-lifecycle-method": "warn",
+        "@angular-eslint/component-class-suffix":"warn",
+        "@angular-eslint/no-output-on-prefix":"warn",
         "@angular-eslint/directive-selector": [
           "error",
```

Comment:
Lets not remove these "errors" all together. Instead, can we change them to "warn"? Then we can create a new PR to fix all of the warnings... followed by turning this back to errors 😄